README.md

```diff
@@ -1,26 +1,107 @@
+This repo hosts the `chocolatey.chocolatey` Ansible Collection.
 
-## Getting Started
+The collection includes the modules required to configure Chocolatey, as well as manage packages on Windows using Chocolatey.
 
-### Installation Instructions
+## Installation and Usage
 
-TBD
+### Installing the Collection from Ansible Galaxy
+
+Before using the Chocolatey collection, you need to install it with the `ansible-galaxy` CLI:
+
+    ansible-galaxy collection install chocolatey.chocolatey
+
+You can also include it in a `requirements.yml` file and install it via `ansible-galaxy collection install -r requirements.yml` using the format:
+
+```yaml
+collections:
+- name: chocolatey.chocolatey
+```
+
+### Modules
+
+This collection provides the following modules you can use in your own roles:
+
+| Name                          | Description                               |
+|-------------------------------|-------------------------------------------|
+|`win_chocolatey`               | Manage packages using chocolatey          |  
+|`win_chocolatey_config`        | Manage Chocolatey config settings         |
+|`win_chocolatey_facts`         | Create a facts collection for Chocolatey  |
+|`win_chocolatey_feature`       | Manage Chocolatey features                |
+|`win_chocolatey_source`        | Manage Chocolatey sources                 |
 
 ### Examples
 
-TBD
+Some example usages of the modules in this collection are below.
+
+Upgrade all packages with Chocolatey:
+
+```yaml
+- name: Upgrade installed packages
+  win_chocolatey:
+    name: all
+    state: latest
+```
+
+Install version 6.6 of `notepadplusplus`:
+
+```yaml
+- name: Install notepadplusplus version 6.6
+  win_chocolatey:
+    name: notepadplusplus
+    version: '6.6'
+```
+
+Set the Chocolatey cache location:
+
+```yaml
+- name: Set the cache location
+  win_chocolatey_config:
+    name: cacheLocation
+    state: present
+    value: C:\Temp
+```
+
+Use Background Mode for Self-Service (C4B):
+
+```yaml
+- name: Use background mode for self-service
+  win_chocolatey_feature:
+    name: useBackgroundService
+    state: enables
+```
+
+Disable the Community Package Repository
+
+```yaml
+- name: Disable Community Repo
+  win_chocolatey_source:
+    name: chocolatey
+    state: disabled
+```
+
+## Testing and Development
+
+If you want to develop new content for this collection or improve what's already here, the easiest way to work on the collection is to clone it into one of the configured [`COLLECTIONS_PATHS`](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths), and work on it there.
+
+### Testing with `ansible-test`
+
+The `tests` directory contains configuration for running integration tests using [`ansible-test`](https://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html).
+
+You can run the collection's test suites with the commands:
+
+    ansible-test integration --docker -v --color
```

Comment:
This should be in a fenced code block. 

Also, don't think this will actually work for this collection. This needs to run with `windows-integration` rather than the standard `integration` mode, and I'm also not 100% sure if the docker container is actually sufficient to run all that.

Might be better to include an example of using the built in test script (though that is PowerShell and not bash) which spins up the Vagrant environment with all necessary prereqs.

Reply:
Thinking on this more... hm. This will _almost_ work once the collection's actually installed. Still need to update the command a bit here, so...

````suggestion
```code
ansible-test windows-integration --docker -v --color
```
````